git-hg/updateforesttags.sh

```diff
@@ -60,91 +61,91 @@ if ! which git-remote-hg 2>/dev/null; then
 fi
 
 # Clone current AdoptOpenJDK repo
-cd $WORKSPACE/$GITHUB_REPO
-echo "Clone current $GITHUB_REPO"
-git clone $GITHUB_PROJECT/$GITHUB_REPO.git
-cd $GITHUB_REPO
+cd "$WORKSPACE"/"$GITHUB_REPO" || exit
```

Comment:
I'd stick a non-zero return code on here - so `exit 1` (Same for a few other currrences later on)

Reply:
Agreed, changing...